doc/examples/onnx_importer.md

```diff
@@ -0,0 +1,88 @@
+# ONNX importer usage examples
+
+This sample demonstrates how to use ONNX importer API.
+The API allows to create nGraph Function from ONNX importer model.
+
+All functions forming ONNX importer API are in [onnx.hpp][onnx_header] header file.
+The available functions can be divided into two categories from user's point of view.
+The first group contains helpers' functions used to check which ONNX op are supported in current version of ONNX importer.
+The second group contains function used to read ONNX models from stream or file. Their result is nGraph function which can be calculated on Inference Engine.
+
+## How to check which ONNX ops are supported
+
+In order to list all supported ONNX ops in a specific version and domain `get_supported_operators` function should be used.
+The code listing below shows how to do it.
+```
```

Comment:
Thanks for this example.  To provide a consistent experience, we don't currently support markdown as documentation for ngraph; can you convert this file to two files:  

(A) One `.rst` file with the "how to" instruction on the API access 
(B) One `.cpp` file that contains just the code you're documenting below?

Should look something like this::
```
.. code-block:: cpp

   const std::int64_t version = 12;
   const std::string domain = "ai.onnx";
   const std::set<std::string> supported_ops = ngraph::onnx_import::get_supported_operators(version, domain);
       for(const auto& op : supported_ops)
       {
              std::cout << op << std::endl;
       }
```